packages/seeds-icons/package.json

```
@@ -27,7 +27,6 @@
     "svgo": "^3.0.0",
     "svgstore": "^3.0.1",
     "ts-node": "^10.9.1",
```

```suggestion
    "ts-node": "^10.9.1"
```
my bad, need to remove this trailing comma also